NAudio.Midi/Midi/MidiIn.cs

```diff
@@ -145,6 +193,19 @@ protected virtual void Dispose(bool disposing)
             if(!this.disposed) 
             {
                 //if(disposing) Components.Dispose();
+
```

Comment:
maybe wrap this block in an if (SysexBufferHeaders.Length > 0) so there is no change to existing behaviour if you're not using sysex

Reply:
Agreed - done. Though the only risk was a failure in midiInClose as it was otherwise guarded by the test of disposed.